docs/source/commands/contracts.mdx

```diff
@@ -0,0 +1,188 @@
+---
+title: Rover contract commands
+description: For use with Apollo Contract variants
+---
+
+A **contract** is a variant that is the result of filtering on a source variant:
+
+Rover commands that interact with contracts begin with `rover contract`.
+
+## Describing a contract variant
```

Comment:
What do you think about putting `contract publish` _before_ `contract describe`? My reasoning is that unless you already have a contract in place, `contract describe` won't work. For folks reading docs on this for the first time it would make more sense to `contract publish` and _then_ `contract describe` in order to see what was just published.

Reply:
yeah i like the idea of putting `publish` before `describe`! i'll reorder it :)